decidim-core/app/serializers/decidim/exporters/serializer.rb

```diff
@@ -54,6 +54,15 @@ def finalize(serialized_data)
       def event_name
         ActiveSupport::Inflector.underscore(self.class.to_s).sub("/", ".serialize.").gsub("/", ".")
       end
+
+      private
+
+      # helper method to serialize taxonomies for any resource
+      def serialize_taxonomies(resource)
```

Comment:
Just to remove noise...
```suggestion
      def taxonomies
```